docs/resources/task.md

```diff
@@ -24,6 +24,7 @@ resource "iterative_task" "example" {
   storage {
     workdir = "."         # default blank (don't upload)
     output  = "results"   # default blank (don't download). Relative to workdir
+    exclude = ["/.dvc/cache", "/results/tempfile", "*.pyc"]
```

Comment:
Absolute paths are rather unintuitive. I'd take relative paths and prepend the initial slash internally, after making sure they're relative.

Reply:
We can't just prepend every exclude with a slash, that would alter the semantics.

Reply:
Isn't the following possible?

```hcl
exclude = [".dvc/cache", "results/tempfile", "**/*.pyc"]
```

See also https://rclone.org/filtering

Reply:
> Isn't the following possible?
> 
> ```terraform
> exclude = [".dvc/cache", "results/tempfile", "**/*.pyc"]
> ```
It is, the example I added includes both absolute and relative paths. And I'd like to keep supporting both, so that users can do something like `exclude = ["/results/tempfile"]`

Reply:
To my mind, the initial `/` is futile and might be misleading. What you specify as `/results/tempfile` would be equivalent to `results/tempfile` (explicitly relative to wirking directory, anchored to it) and what you specify as `*.pyc` would be equivalent to `**/*.pyc` (again, explicitly relative, but not anchored)

Reply:
> To my mind, the initial `/` is futile and might be misleading. What you specify as `/results/tempfile` would be equivalent to `results/tempfile` (explicitly relative to working directory, anchored to it) and what you specify as `*.pyc` should[1](#user-content-fn-1-4afe5f030dbddebf9f7cbb0362d0c8d2) be equivalent to `**/*.pyc` (again, explicitly relative, but not anchored)

ok, so `results/tempfile` would internally be converted to `/results/tempfile`? What about `a.txt`? Because the behavior would change - an exclude pattern `a.txt` means "ignore files named a.txt wherever in the directory tree they are, and `/a.txt` means ignore a.txt in the root directory only.

This pattern is uniform acress `.*ignore` files and might be something our users are already aware of.

Reply:
So what exactly do you suggest?

Reply:
There are trade-offs involved here - if we take all patterns to be relative to `workdir` (by silently prepending `/`), user's will have to add `**.pyc` to ignore .pyc files regardless of their location (instead of `*.pyc`).

Reply:
Indeed, and none of the options seems clearly better to me; I'll defer to @iterative/cml for opinions